tests/e2e/lib/pages/wp-admin/block-editor.js

```diff
@@ -16,9 +16,9 @@ export default class BlockEditorPage extends Page {
 		super( page, { expectedSelector, url } );
 	}
 
-	async insertBlock( blockName ) {
-		await searchForBlock( blockName );
-		const blockIconSelector = `.editor-inserter__menu button[aria-label*='${ blockName }']`;
+	async insertBlock( blockName, blockTitle ) {
+		await searchForBlock( blockTitle );
+		const blockIconSelector = `.editor-block-list-item-jetpack-${ blockName }`;
```

Comment:
Gutenberg removed the `aria-label` property, and changed their equivalent selector in tests to an XPath selector. I found this to be unreliable, and felt fragile when applied to beta blocks, which have the "Beta" term in their displayed title. It's much more reliable to use the unique name that block defines.